angular-cli.json

```diff
@@ -20,9 +20,7 @@
       "styles": [
         "styles.css"
       ],
-      "scripts": [
-        "../node_modules/jquery/dist/jquery.min.js"
-      ],
+      "scripts": [],
```

Comment:
Can you please explain exactly why this line had to be removed, I am still a little confused